float64 leg_pos_kp
float64 leg_pos_kd
float64 leg_for_kp
float64 leg_for_kd
float64 hip_pos_kp
float64 hip_pos_kd
float64 left_leg_len
float64 right_leg_len
float64 left_leg_ang
float64 right_leg_ang
float64 left_fx
float64 right_fx
float64 left_fz
float64 right_fz
float64 left_ampx
float64 right_ampx
float64 left_ampz
float64 right_ampz
float64 left_freqx
float64 right_freqx
float64 left_freqz
float64 right_freqz
float64 left_offx
float64 right_offx
float64 left_offz
float64 right_offz

uint8 left_controller
uint8 right_controller
